core/smartcontract_analyser.go

```diff
@@ -0,0 +1,302 @@
+package core
```

Comment:
Few pointers:

- The functions present here would be standalone instead of being methods of `Core` struct.
- Instead of logging, the messages should be returned from the function

Reply:
All the methods apart from the AnalyseCode have been removed from the ```Core``` struct